app/assets/javascripts/govuk_publishing_components/lib/shim-links-with-button-role.js

```diff
@@ -0,0 +1,36 @@
+// migrated from govuk_frontend_toolkit
+
+// javascript 'shim' to trigger the click event of element(s)
```

Comment:
We don't need to migrate this one as it's covered by [button.js in govuk-frontend](https://github.com/alphagov/govuk-frontend/blob/master/src/govuk/components/button/button.js#L21-L30).